docs/ReferenceChanges.md

```diff
@@ -0,0 +1,41 @@
+Package: Microsoft.Extensions.Configuration.Binder
+  TFM: .NETStandard2.0
+    Removed:
+      Microsoft.Extensions.Configuration
+Package: Microsoft.Extensions.Configuration.Json
+  TFM: .NETStandard2.0
+    Removed:
+      System.Threading.Tasks.Extensions
+Package: Microsoft.Extensions.Hosting.Abstractions
+  TFM: .NETCoreApp5.0
+    Removed:
+      Microsoft.Extensions.Logging.Abstractions
+  TFM: .NETStandard2.0
+    Removed:
+      Microsoft.Extensions.Logging.Abstractions
+Package: Microsoft.Extensions.Logging
+  TFM: .NETCoreApp5.0
+    Removed:
+      Microsoft.Extensions.Configuration.Binder
+  TFM: .NETStandard2.0
+    Removed:
+      Microsoft.Extensions.Configuration.Binder
+Package: Microsoft.Extensions.Logging.Console
+  TFM: .NETCoreApp5.0
+    Removed:
+      Microsoft.Extensions.Configuration.Abstractions
+  TFM: .NETStandard2.0
+    Removed:
+      Microsoft.Extensions.Configuration.Abstractions
+Package: Microsoft.Extensions.Logging.EventLog
+  TFM: .NETFramework4.6.1
+    Removed:
+      System.Diagnostics.EventLog
```

Comment:
Is this because the type is just part of .NET Framework 4.6.1? (@ericstj ?)

Reply:
Yes